api/control.proto

```diff
@@ -191,6 +191,8 @@ message ListTasksRequest {
 		repeated docker.swarmkit.v1.TaskState desired_states = 6;
 		// NamePrefixes matches all objects with the given prefixes
 		repeated string name_prefixes = 7;
+		repeated string runtimes = 9;
```

Comment:
Do you think it makes sense to make this `TaskSpec.runtime` instead of a string, so we can select by type instead of using a string describing the type?

I'm not sure it makes sense, but just wanted to share the idea.